autospec/specdescription.py

```diff
@@ -242,6 +242,8 @@ def scan_for_description(package, dirn):
     """Scan the project directory for things we can use to guess a description and summary."""
     for dirpath, _, files in os.walk(dirn):
         for name in files:
+            if name.lower().endswith(".pdf"):
+                break
```

Comment:
I think this should be a continue rather than a break otherwise looks good.

Reply:
Ah, yeah I agree.